frontend/src/components/ProvidersDrawer/ProvidersDrawer.tsx

```diff
@@ -11,13 +11,15 @@ import { CSVUploader } from '../CSVUploader/CSVUploader'
 export let ProvidersDrawer = withStore('uploadedProvidersFilename')(({ store }) =>
   <div>
     <h2>Providers</h2>
-    <CSVUploader onUpload={onFileSelected(store)} />
+    <div className='Flex -Row'>
+      <CSVUploader className='-Flex-0' onUpload={onFileSelected(store)} />
+      {store.get('uploadedProvidersFilename') && <ClearInputsButton onClearInputs={onClearInputs(store)} />}
+    </div>
     <p className='Ellipsis Muted SmallFont'>{
```

Comment:
Should we create a class for this type of behavior?

Reply:
Maybe eventually, but I didn't think it's necessary yet. The fact that the `Ellipsis`, `Muted`, and `SmallFont` make a single class less necessary, because it's so easy to compose those utils to get the desired behavior.